docs/plugins/purchase.md

```diff
@@ -88,72 +84,32 @@ Purchase.setup('http://csp-backend.com/api', {
     uri: '/subscriptions/:id',
     method: 'DELETE',
   },
-  signature: {
-    uri: '/assets/:id/signature',
-    method: 'POST',
-    headers: {
-      'x-auth-token': 'abcd123xyz'
-      'cache-control': 'no-cache'
-    }
-  },
 })
 ```
 
-In some cases the CSP backend setup is too different from the default specification, that it becomes too difficult
-to configure the enpoint uri's. In those cases you can specify a `callback` instead, and handle the request to the
-CSP backend entirely yourself.
+> If the CSP backend does *not* completely follow the expected blueprint, or can not be configured properly via the `setup` method, the steps in the `buy` method can also be implemented *manually*. In that case, you can use the Plugin methods `signature`, `payment`, `subscribe` and `confirm` (see [below](#signature)).
 
-The callback function will receive 2 arguments: `params` and `data` and should _always_ return a `Promise` which resolves the result (or rejects an error in case of a failure). When a callback is specified, all other keys in that `endpoint` config
-are ignored.
```

Comment:
Info about callbacks might have been added afterwards and got lost now

Reply:
Info added.

Reply:
I also noticed that the sentence "If the CSP backend [...]" was placed in the Setup section, where it should be in the Buy section. Sentence moved to the Buy section.